include/bluetooth/mesh/light_ctl_srv.rst

```diff
@@ -0,0 +1,128 @@
+.. _bt_mesh_light_ctl_srv_readme:
+
+Light CTL Server
+######################
+
+The Light Color Tunable Light (CTL) Server represents a single light on a mesh device.
+It should be instantiated in the light fixture node.
+
+The Light CTL Server adds the following new model instances in the composition data, in addition to the extended :ref:`bt_mesh_lightness_srv_readme` model:
+
+* Light CTL Server
+* Light CTL Setup Server
+
+These model instances share the states of Light CTL Server, but accept different messages.
+This allows for a fine-grained control of the access rights for the Light CTL states, as these model instances can be bound to different application keys:
+
+* The Light CTL Server provides write access to the Lightness, Temperature and Delta UV states for the user, in addition to read access to all the meta states.
+* The Light CTL Setup Server provides write access to the Default CTL state and Temperature Range meta states.
+  This allows the configurator devices to set up the Temperature range and the default CTL state.
+
+In addition to the extended Lightness Server model the Light CTL Server is also assosiated with an instance of the :ref:`bt_mesh_light_temp_srv_readme` model.
+This model instance must be located on the same device, where the element address of the Light Temperature Server must be greater than the address of the
+Light CTL Server.
+
+States
+======
+
+The Generic Power OnOff Server model (extended by the Lightness Server model) contains the following states:
+
+Lightness: ``uint16_t``
+    The Lightness state represents the emitted light level of an element, and ranges from ``0`` to ``65535``.
+    The Lightness state is bound to the Light Lightness Actual state of the extended :ref:`bt_mesh_lightness_srv_readme`:
+
+    .. code-block:: console
+
+       Light CTL Lightness = Light Lightness Actual
+
+    The Lightness state power up behavior is determined by the On Power Up state of the extended :ref:`bt_mesh_ponoff_srv_readme`:
+
+    * :cpp:enumerator:`BT_MESH_ON_POWER_UP_OFF <bt_mesh_ponoff::BT_MESH_ON_POWER_UP_OFF>` - The Lightness state is set to ``0`` on power up.
+    * :cpp:enumerator:`BT_MESH_ON_POWER_UP_ON <bt_mesh_ponoff::BT_MESH_ON_POWER_UP_ON>` - The Lightness state is set to Default Lightness on power up.
+    * :cpp:enumerator:`BT_MESH_ON_POWER_UP_RESTORE <bt_mesh_ponoff::BT_MESH_ON_POWER_UP_RESTORE>` - The Lightness state is set to the last known Light level (zero or non-zero).
```

Comment:
These enumerator references should be changed when https://github.com/nrfconnect/sdk-nrf/pull/2789 is merged

Reply:
Will do it as soon as the PR is merged

Reply:
I have notified them 👍